pkg/run/run.go

```diff
@@ -44,14 +44,15 @@ func (b *DockerRunner) Run(config *api.Config) error {
 
 	opts := docker.RunContainerOptions{
 		Image:        config.Tag,
+		Entrypoint:   []string{"/bin/env"},
```

Comment:
>make it a default where?

In the methods that are using this (docker.CommitContainer/docker.RunContainer).